Add note about RHEL and Debian images (#1531)
n8n-io/n8n-docs: docs/1-0-migration-checklist.md

docker run --rm -it --user root -v ~/.n8n:/home/node/.n8n --entrypoint chown n8nio/base:16 -R node:node /home/node/.n8n
```

> We have also removed the Debian and RHEL images, If you were previously using these you will need to change the image being used which shouldn't result in any errors unless you were making a custom image based on one of those images. 

Reviewer: ```suggestion
We have also removed the Debian and RHEL images. If you were using these you need to change the image you use. This shouldn't result in any errors unless you were making a custom image based on one of those images. 
```
